src/app/components/page/PublicPage.ts

```diff
@@ -0,0 +1,16 @@
+import { useEffect, ReactChildren } from 'react';
+
+type Props = {
+    children: ReactChildren;
```

Comment:
Not the correct type for children, use `React.ReactNode`

`ReactChildren` is an interface for the util class: https://reactjs.org/docs/react-api.html#reactchildren